lib/cryptography/utils.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Kadena.Cryptography.Utils do
+  @moduledoc """
+  Documentation for `Cryptography` utils functions.
+  """
+  @type hash :: String.t()
+  @type str :: String.t()
+  @type key :: String.t()
+  @type opts :: Keyword.t()
+
+  @spec bin_to_hex(key_bin :: binary()) :: String.t()
+  def bin_to_hex(key_bin), do: Base.encode16(key_bin, case: :lower)
+
+  @spec hex_to_bin(key :: String.t()) :: binary()
```

Comment:
We could be more specific in Key type, for instance, Kadena.Type.Base16String

Reply:
Agree!